pkg/api/pod/util_test.go

```diff
@@ -1757,3 +1757,100 @@ func TestDropOSField(t *testing.T) {
 		}
 	}
 }
+
+func TestDropPodStatusHostIPFields(t *testing.T) {
+	podWith2IPInHostIP := func() *api.Pod {
+		return &api.Pod{
+			Status: api.PodStatus{
+				HostIPs: []api.HostIP{
+					{
```

Comment:
a) doesn't seem to be gofmt'ed

b) This is a LOT of whitespace.

Can  you instead say:

```
		return &api.PodStatus{
				HostIPs: []api.HostIP{{
						IP: "10.0.0.1",
					}, {
						IP: "fd00:10::1",
					}},
			}
```

or even:

```
		return &api.PodStatus {
			HostIPs: makeHostIPs("10.0.0.1", "fd00:10::1")
		}
```

with

```
func makeHostIPs(ips ...string) []api.HostIP {
	ret := []api.HostIP{}
	for _, ip := range ips {
		ret = append(ret, api.HostIP{ IP: ip})
	}
	return ret
}
```